packages/server-wallet/src/db/migrations/20201014174039_open-channel-objectives.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+
+const tableName = 'open-channel-objectives';
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable(tableName, function(table) {
+    table.integer('objectiveId');
```

Comment:
This should be a primary key.

Reply:
I made that change, but rolled it back. We need to first make sure `objectiveId` is unique.

Reply:
What makes it not unique? That suggests an error in the client code to me, but I don't know if I have the right context

Reply:
Yeah it's definitely a problem in the client code. We can probably just have the db choose an incrementing id for us (currently it is just a placeholder: the `channelNonce`, in fact).